### DISCUSSION TITLE
zepeto 카메라를 1인칭으로 어떻게 변경할 수 있을까요?

### DISCUSSION THREAD

**Discussion by JaeHyeon774** [naverz/zepeto-studio-kor#659] [5. Q&A]
zepeto studio에서는 3인칭으로 밖에 볼 수 없는데 1인칭으로 어떻게 변경할 수 있을까요?

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요. 1인칭은  카메라의 줌 속성을 바꿔서 손 쉽게 구현이 가능합니다. 

ZepetoPlayers > Camera 속성을 
min, max를 0.1 정도의 작은 숫자로 변경 후 플레이하면 1인칭이 됩니다.

<img width="436" alt="다운로드" src="https://user-images.githubusercontent.com/100118650/165206747-c69efb24-0e5a-4b36-8e5a-ba4f7bd6f99b.png">

https://studio.zepeto.me/kr/guides/character-control

​카메라 설정 관련 가이드를 참고하여 주세요.

감사합니다.

**Reply by JaeHyeon774** [reply]
감사합니다.
![image](https://user-images.githubusercontent.com/55771437/165209922-12d2865a-862c-4a7f-805b-f2b52ab2dc94.png)
그런데 이렇게 위에서 내려다 보이는게 아니라 zepetoplayer가 바라보는 방향대로 카메라가 따라다니게 하려면 카메라를 추가해야하는 건가요?

**Reply by x4cf** [reply]
Look Offset 을 0,0,0 으로 해보세요 :)

**Reply by JaeHyeon774** [reply]
캐릭터가 눈으로 바라보는 느낌이 아니라 이렇게 나옵니다...ㅜㅠ
![image](https://user-images.githubusercontent.com/55771437/165222070-52a694d6-c091-4f77-96b4-7ae1ebac76cf.png)
제페토 어플로 월드에 들어가면 1인칭 3인칭 지정하는게 있는데 그런 느낌을 줄 수 있을까 해서요

**Comment by hyunjoon-park-metaspace** [comment]
카메라 Min, Max 값 두 개를 `-0.85`로 맞춰보세요

**Reply by JaeHyeon774** [reply]
감사합니다!!
